tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -5,53 +5,53 @@ weave_options:
 $$
 
-where $\Sigma_{d \times d}$ contains only $r$ (where $r<=min(m,n)$ is the rank of $A$) non-zero singular values. If we pad $\Sigma$ with zeros and add arbitrary orthonormal columns to $U$ and $V$, we obtain the more compact  orm [1]:
+where $\Sigma_{r \times r}$ contains only $r$ (where $r<=min(N,D)$ is the rank of $A$) non-zero singular values of $\mathbf{X}$. If we pad $\Sigma$ with zeros and add arbitrary orthonormal columns to $U$ and $V$, we obtain the more compact form [1]:
 
 $$
-A_{m \times n} =  U_{m \times m} \Sigma_{m \times n}  V_{n \times n}^T
+\mathbf{X}_{N \times D} = \mathbf{U}_{N \times N} \mathbf{\Sigma}_{N \times D} \mathbf{V}_{D \times D}^T
 $$
 
-where $U$ and $V$ are unitary matrices (i.e. with orthonormal columns). Such decomposition always exists for any complex matrix. columns of $V$ are the principal directions/axes. Percentage of variations explained can be calculated using the ratios of singular values.
+where $U$ and $V$ are unitary matrices (i.e. with orthonormal columns). Such decomposition always exists for any matrix. Columns of $V$ are the principal directions/axes. Percentage of variations explained can be calculated using the ratios of singular values [3].
 
-Here we take a probabilistic perspective. For more details and a mathematical derivation, we recommend Bishop's textbook (Christopher M. Bishop, Pattern Recognition and Machine Learning, 2006). The idea of proabilistic PCA is to find a latent variable $z$ that can be used to describe the hidden structure in a dataset.
+Here we take a probabilistic perspective. For more details and a mathematical derivation, we recommend Bishop's textbook (Christopher M. Bishop, Pattern Recognition and Machine Learning, 2006). The idea of proabilistic PCA is to find a latent variable $z$ that can be used to describe the hidden structure in a dataset [2]. Consider a data set $X_{D \times N}=\{x_i\}$ with $i=1,2,...,N$ data points, where each data point $x_i$ is D-dimensional (i.e. $x_i \in \mathcal{R}^D$). Note that, here we use the flipped version of the data matrix. We aim to represent the original $n$ dimensional vector using a lower dimensional a latent variable $z_i \in \mathcal{R}^k$.
 
-# The model [2]
+We first assume that each latent variable $z_i$ is normally distributed:
 
-Consider a data set $X={x_i}$  with $i=1,2,...,m$ data points, where each data point $x_i$ is n-dimensional (i.e. $x_i \in \mathematical{R}^n$). We aim to represent the original $n$ dimensional vector using a lower dimensional a latent variable $z_i \in \mathcal(R)^k$.
-
-We first assume that each latent variable is normally distributed:
 $$
 z_i \sim \mathcal{N}(0, I)
 $$
-and the corresponding data point is generated via projection
+
+and the corresponding data point is generated via projection:
+
 $$
-x_i | z_i = \mathcal{N}(x_i | W z_i + \mu, \sigma^2 I)
+x_i | z_i= \mathcal{N}(x_i | W z_i + μ, \sigma^2 I)
```

Comment:
```suggestion
x_i | z_i= \mathcal{N}(x_i | \mathbf{W} z_i + \bolsymbol{μ}, \sigma^2 \mathbf{I})
```

This is an identity matrix. So $I$ should be $\mathbf{I}$
And $W$ should be $\mathbf{W}$

$\mu$ is a vector? So it should be $\boldsymbol{\mu}$

Reply:
yes, absolutely correct - I just changed all μ to be $\boldsymbol{\mu}$. Also $\mathbf{I}$ and $\mathbf{W}$.